### PR TITLE
Updates to ODF latest-stable-4.10.0 image (cypress)

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
https://issues.redhat.com/browse/RHSTOR-2414

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts [hunk 1]
```diff
@@ -1,55 +1,107 @@
...
+  cy.wait(30000);
+  linkPullSecretToPods();
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(30000);
+  linkPullSecretToPods(false);
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(120000);
+  linkPullSecretToPods(false);
+  // eslint-disable-next-line cypress/no-unnecessary-waiting
+  cy.wait(120000);
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts:42]: Too many arbitrary waits. I would rather we poll on something.

**Review by bipuladh — CHANGES_REQUESTED** [review]

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts (hunk 1 above)
Comment (SanjalKatiyar) [reply frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts:42]: I copied from here: https://github.com/red-hat-storage/odf-console/blob/9cd8ecd5a50023d0058165574ae4591680d549b1/test-prow-e2e.sh#L90

Earlier as well we were waiting: https://github.com/openshift/console/blob/108daa3c51158e6fef665da1834d6d7a933f9e18/frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/index.ts#L38 I just added few more wait as per odf-console's script.

**Pull request closed without merging**